db/view/view_update_generator.cc

```diff
@@ -24,25 +24,28 @@
 namespace db::view {
 
 future<> view_update_generator::start() {
-    _started = seastar::async([this]() mutable {
-        while (!_as.abort_requested()) {
```

Comment:
It's somewhat nicer to provide a thread_attributes object with sched_group set correctly.

Reply:
Neat - done.